Package.swift

```diff
@@ -17,18 +17,20 @@ import PackageDescription
 
 let package = Package(
     name: "async-http-client",
+    platforms: [.iOS(.v12), .tvOS(.v12)],
     products: [
         .library(name: "AsyncHTTPClient", targets: ["AsyncHTTPClient"]),
     ],
     dependencies: [
         .package(url: "https://github.com/apple/swift-nio.git", from: "2.13.1"),
         .package(url: "https://github.com/apple/swift-nio-ssl.git", from: "2.4.1"),
         .package(url: "https://github.com/apple/swift-nio-extras.git", from: "1.3.0"),
+        .package(url: "https://github.com/adam-fowler/swift-nio-transport-services.git", .branch("master")),
```

Comment:
these changes need to be upstreamed

Reply:
There is a conversation about his already on https://github.com/apple/swift-nio-transport-services/pull/74. Looks like we'll be changing how this is done

Reply:
Using swift-nio-transport-services 1.5.1